api/actions/actionsRouter.js

```diff
@@ -1,8 +1,116 @@
+ *        resolved:
+ *          type: boolean
+ *          description: Resolved case set to false from start
+ *        strike:
+ *          type: boolean
+ *          description: Need more clarification on this will update soon
+ *        created_at:
+ *          type: timestamp
+ *          description: Automatic date-time string from a action's creation in the database
+ *        updated_at:
+ *          type: timestamp
+ *          description: Automatic date-time string from a action's last update in the database
+ *      example:
+ *        action_ticket_id: 1
+ *        submitted_by: 7
+ *        subject_id: 10
+ *        issue: "Spencer missed his 2nd weekly session, may be dropped?"
+ *        comments: null
+ *        pending: true
+ *        resolved: false
+ *        strike: true
+ *        created_at: "2022-01-26 15:33:34.945832-07"
+ *        updated_at: "2022-01-26 15:33:34.945832-07"
+ */
 
+/**
+ * @swagger
+ * /actions:
+ *  get:
+ *    summary: Get the list of all action tickets
+ *    description: Provides a JSON array of action tickets (as objects)
+ *    tags:
+ *      - action
+ *    security:
+ *      - okta: []
+ *    responses:
+ *      '200':
+ *        description: An array of action objects
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: array
+ *              items:
+ *                $ref: '#/components/schemas/Actions'
+ *              example:
+ *                - action_ticket_id: 1
+ *                  submitted_by: 7
+ *                  subject_id: 10
+ *                  issue: "Spencer missed his 2nd weekly session, may be dropped?"
+ *                  comments: null
+ *                  pending: true
+ *                  resolved: false
+ *                  strike: true
+ *                  created_at: "2022-01-26 15:33:34.945832-07"
+ *                  updated_at: "2022-01-26 15:33:34.945832-07"
+ *                - action_ticket_id: 1
+ *                  submitted_by: 7
+ *                  subject_id: 10
+ *                  issue: "Spencer missed his 2nd weekly session, may be dropped?"
+ *                  comments: null
+ *                  pending: true
+ *                  resolved: false
+ *                  strike: true
+ *                  created_at: "2022-01-26 15:33:34.945832-07"
+ *                  updated_at: "2022-01-26 15:33:34.945832-07"
+ *                - action_ticket_id: 1
+ *                  submitted_by: 7
+ *                  subject_id: 10
+ *                  issue: "Spencer missed his 2nd weekly session, may be dropped?"
+ *                  comments: null
+ *                  pending: true
+ *                  resolved: false
+ *                  strike: true
+ *                  created_at: "2022-01-26 15:33:34.945832-07"
+ *                  updated_at: "2022-01-26 15:33:34.945832-07"
+ *      '401':
+ *        $ref: '#/components/responses/UnauthorizedError'
+ *      '403':
```

Comment:
I don't think we actually have any 403 errors at the moment; I know this was referenced from another router's docs, but I suggest removing this, unless you can verify there are indeed 403 errors, and that they are actually returning the same message as the unauthorized error.